keyboards/setta21/rev1/rev1.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+
```

Comment:
This doesn't compile for me without:

```suggestion
#include "setta21.h"
```

Reply:
That's...odd. setta21.h should be (and is) including *this* file, not the other way around.

Reply:
My best guess at this, without having actually investigated.

- `keyboards/setta21/rules.mk` sets `DEFAULT_FOLDER = setta21/rev1`
- `make` kicks over to `setta21/rev1`, and starts building
- `keyboards/setta21/rev1/rev1.c` includes `keyboards/setta21/rev1/rev1.h`
- `keyboards/setta21/rev1/rev1.h` doesn't include the main `keyboards/setta21/setta21.h`, which has the `quantum.h` include, so `matrix_init_user()` isn't defined.